src/javascript/app/common/form_validation.js

```diff
@@ -346,13 +347,17 @@ const Validation = (() => {
             }
 
             if (!field.is_ok) {
-                message = options.message || ValidatorsMap.get(type).message;
+                let message_template;
+                field.$error.text(options.message || ValidatorsMap.get(type).message);
                 if (type === 'length') {
-                    message = template(message, [options.min === options.max ? options.min : `${options.min}-${options.max}`]);
+                    message_template = template(message, [options.min === options.max ? options.min : `${options.min}-${options.max}`]);
```

Comment:
Comparing to the previous version, on this stage the message already had a value `options.message || ValidatorsMap.get(type).message` but now you will always pass an empty constant. Doesn't it change the message content?

Reply:
The message constant is not needed anymore, I removed it. Instead, I just pass an empty string in the template functions now.